apps/coscrad-frontend/src/configurable-front-matter/frontMatterData/FrontMatter.ts

```diff
@@ -0,0 +1,18 @@
+import { NonEmptyString } from '@coscrad/data-types';
+
+export default class FrontMatter {
```

Comment:
We never actually used this in our previous work, but left it as a placeholder. I recommend you do the same as I suspect major refactors are coming.

Reply:
This is gone now in favour of a functional approach to essentially the same thing.